ixbrlparse/core.py

```diff
@@ -177,6 +177,11 @@ def _get_nonnumeric(self) -> None:
                         value=text.strip().replace("\n", "")
                         if isinstance(text, str)
                         else "",
+                        ixt=self.namespaces.get(
+                            f'xmlns:{s.get("format").split(":")[0]}', []
+                        )[0]
+                        if s.get("format") is not None
+                        else None,
```

Comment:
This needs to be a bit more robust for the case where it's not found in `self.namespaces` otherwise we'll get lots of keyerrors.